api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -862,75 +871,46 @@ def drop_tip(
         home_after: Optional[bool] = None,
     ) -> InstrumentContext:
         """
-        Drop the current tip.
+        Drop the current tip. See :ref:`pipette-drop-tip`.
 
-        If no location is passed, the Pipette will drop the tip into its
-        :py:attr:`trash_container`, which if not specified defaults to
-        the fixed trash in slot 12.  From API version 2.15 on, if the trash container is
-        the default fixed trash in A3 (slot 12), the API will default to
-        dropping tips in different points within the trash container
-        in order to prevent tips from piling up in a single location in the trash.
+        If no location is passed (e.g. ``pipette.drop_tip()``), the pipette will drop the
+        attached tip into its default :py:attr:`trash_container`. For the Flex and OT-2 the
+        default slots for their in-deck trash containers are A3 and 12, respectively.
```

Comment:
i wouldn't characterize the OT-2 as having an "in-deck" trash container.
```suggestion
        default slots for their trash containers are A3 and 12, respectively.
```